maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -504,6 +504,13 @@ private void setupDatabaseForServer(Server server) throws MojoExecutionException
 
 		if (server.isMySqlDb() || server.isPostgreSqlDb()) {
 			String uri = getUriWithoutDb(server);
+			if(server.isMySqlDb()) {
+				try {
```

Comment:
Instead of doing this, i would rather put `DriverManager.getConnection`, in DBConnector, in an exception hander, and call it again while catching the thrown SQLException. Something like: https://pastebin.com/

Reply:
I updated the PR. This feels little hacky though :sweat_smile:

Reply:
Can you think of a non hacky alternative? 😊

Reply:
we could do something like this: https://gist.github.com/wikumChamith/1303cfeebef84c91fb21d11fca5f02f3

Reply:
Is it cleaner that the catching of the SqlException in DBConnector?

Reply:
That feels like flipping a switch on and off, to fix the problem. 😂

Reply:
Do you think you can find the underlying cause of this problem? 😊